city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,212 @@
+                meeting["classification"] = NOT_CLASSIFIED
+            return meeting
+
+        def _parse_start(self, response):
+            """Parse start datetime as a naive datetime object."""
+            return datetime.strptime(
+                " ".join(
+                    [
+                        i.strip()
+                        for i in response.xpath(
+                            "//div[@class='jev_evdt_header']/div/p/text()"
+                        ).extract()
+                    ]
+                ),
+                "%A, %B %d, %Y %I:%M%p",
+            )
+
+        def _parse_all_day(self, response):
+            """Parse or generate all-day status. Defaults to False."""
+            return False
+
+        def _parse_location(self, response):
+            """Parse or generate location."""
+            addr = re.split(
+                "-|,",
+                response.xpath("//div[@class='jev_evdt_location']/text()")
+                .extract()[0]
+                .strip(),
+                maxsplit=1,
+            )
+            return {
+                # Using reverse indexing for the cases
+                # where there is no building name or no location
+                "address": addr[-1].strip(),
+                "name": addr[0].strip(),
+            }
+
+        def _parse_links(self, response):
+            """Parse or generate links."""
+            return [
+                {"href": item, "title": "Zoom Link"}
+                for item in response.xpath(
+                    "//div[@class='jev_evdt_desc']/p/a/@href"
+                ).extract()[:1]
+            ]
+
+        def _parse_source(self, response):
+            """Parse or generate source."""
+            return response.url
+
+        def _get_status(self, meeting):
+            if "cancelled" in meeting["title"].lower():
+                meeting["status"] = CANCELLED
+            elif datetime.now() > meeting["end"]:
+                meeting["status"] = PASSED
+            else:
+                meeting["status"] = TENTATIVE
+
+            return meeting
+
+    class ChiOhareNoiseSubSpider2:
+        def parse(self, response):
+            """
+            `parse` should always `yield` Meeting items.
+
+            Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+            needs.
+            """
+            for item in response.css("tr.cat-list-row0, tr.cat-list-row1"):
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    start=self._parse_start(item),
+                    links=self._parse_links(item, response),
+                    source=self._parse_source(response),
+                )
+
+                meeting = self._get_status(meeting)
+                meeting = self._parse_classification(meeting)
+                yield meeting
+            next_page = response.xpath("//li[@class='pagination-next']/a/@href").get()
+            if next_page is not None:
```

Comment:
Same comment as above, we can limit this to only getting the next page once if there's a `_parse_page` method that is called here instead of running all of the logic in `parse`